src/Data/Nat/Base.agda

```diff
@@ -335,7 +335,7 @@ suc n ! = suc n * n !
 -- _≤′_: this definition is more suitable for well-founded induction
 -- (see Data.Nat.Induction)
 
-infix 4 _≤′_ _<′_ _≥′_ _>′_
+infix 4 _<′_ _≤′_ _≥′_ _>′_
```

Comment:
This is spurious... don't know how it crept in! Will fix later.
Mind you, it draws attention to the fact that `_≤′_` *also* violates the 'no non-linear constructor' proposal in #2519 ...

Reply:
Pushed.